packages/components/tag/src/tag-body.tsx

```diff
@@ -99,6 +99,9 @@ const TagBody = (props: TTagBodyProps) => {
                 right: -1px;
                 content: '';
                 background-color: ${designTokens.borderColorForTagWhenFocused};
+                background-color: ${props.type === 'warning'
+                  ? designTokens.borderColorForTagWarning
+                  : designTokens.borderColorForTagWhenFocused};
```

Comment:
Thanks for the fix 🎉 A this fixes the bug, I just wanted to point out that the `border-color` is duplicated

Reply:
Ouch! I forgot to remove the previous one 🤦 
Thanks for noticing it 👍 

Removed in this commit: 57c0076f15e130aa4916bdf209c525da94f9669a